src/vorta/views/archive_tab.py

```diff
@@ -544,11 +595,22 @@ def bmountarchive_refresh(self, icon_only=False):
             if not icon_only:
                 self.bMountArchive.setText(self.tr("Unmount"))
                 self.bMountArchive.setToolTip(self.tr('Unmount the selected archive from the file system'))
+                self.bMountArchive.setMenu(None)
+                try:
+                    self.bMountArchive.clicked.disconnect(self.bmountarchive_clicked)  # avoid race condition
```

Comment:
Would `bmountarchive_clicked` be connected to the signal multiple times?
This also doesn't disconnect the slot `lambda: self.bmountarchive_clicked(quick=True)` afaik.